src/librbd/Operations.cc

```diff
@@ -358,6 +359,60 @@ void Operations<I>::execute_rebuild_object_map(ProgressContext &prog_ctx,
 }
 
 template <typename I>
+int Operations<I>::check_object_map(ProgressContext &prog_ctx) {
+  CephContext *cct = m_image_ctx.cct;
+  ldout(cct, 5) << this << " " << __func__ << dendl;
+  int r = m_image_ctx.state->refresh_if_required();
+  if (r < 0) {
+    return r;
+  }
+
+  r = invoke_async_request("check object map", true,
+                           boost::bind(&Operations<I>::check_object_map, this,
+                                       boost::ref(prog_ctx), _1),
+			   [] (Context *c) { c->complete(-EOPNOTSUPP); });
```

Comment:
We discussed this during standup at one point; since this is more of a debugging tool than a useful operation in itself, it was thought that users would want to use rebuild in that case. Also, there is not necessarily the expectation of consistency during active I/O, so it's likely there would be false positives.

Also, I wanted to be the first to commit a C++11 lambda.